lib/blocs/cryptocurrency_bloc.dart

```diff
@@ -0,0 +1,39 @@
+
+import 'dart:async';
```

Comment:
This is the only file that is really confusing to me 😄 Can you provide some resources about this "bloc" thingy? 😄

Reply:
Google: "A variation of this classical pattern has emerged from the Flutter community – BLoC. BLoC stands for Business Logic Components. The gist of BLoC is that everything in the app should be represented as stream of events: widgets submit events; other widgets will respond. BLoC sits in the middle, managing the conversation."

Resources: https://www.youtube.com/watch?v=fahC3ky_zW0&feature=youtu.be

Reply:
Thank you! 🎉